src/output-json.h

```diff
@@ -50,6 +50,7 @@ typedef struct JsonAddrInfo_ {
     Port sp;
     Port dp;
     char proto[JSON_PROTO_LEN];
+    bool log_port;
```

Comment:
Cannot it be within a reduced scope, not inside a struct, but just as a variable in `CreateEveHeader` where we have the Packet as argument ?

Reply:
Yes, it could be an alternative.